xm-commons-lep/src/main/java/com/icthh/xm/commons/lep/XmLepLoggingExecutorListener.java

```diff
@@ -49,19 +77,25 @@ private void onAfterEvent(AfterResourceExecutionEvent afterEvent) {
         if (exception.isPresent()) {
             logStopError(signatureToPrint, scriptName, exception.get());
         } else {
-            logStop(signatureToPrint, scriptName);
+            LepLogConfiguration loggingConfig = loggingConfigService.getLepLoggingConfig(afterEvent.getKey().getId());
```

Comment:
are you sure there is no chance to get NPE on afterEvent.getKey().getId() ?

Reply:
used scriptName, and added null check to getLepLoggingConfig